### ISSUE TITLE
[FR] CKEditor Support

### ISSUE THREAD

**Issue by kylecotter** [venveo/craft-redactor-split#6]
More of a "plugin idea" than feature request, but curious if a CKEditor version of this plugin is on your roadmap since that's what Pixel & Tonic is pushing now for WYSIWYG?

Thanks!

**Comment by Mosnar** [comment]
Good call! I haven't spent any time with CKEditor yet really, but how hard could it be, right?
 🤔
